examples/ed25519_signature.ts

```diff
@@ -0,0 +1,42 @@
+import { Balances, users } from "westend_dev/mod.js"
+import { MultiAddress } from "westend_dev/types/sp_runtime/multiaddress.js"
+import * as ed from "../deps/ed25519.ts"
```

Comment:
Since this is only a dependency of the example, let's inline the url here and delete `deps/ed25519.ts`.